### PR TITLE
Do not require watched groups to have view permission on items in activitiesView & skillsView

### LINKED ISSUE #738: Do not require the watched group to view the item to list it in activitiesView
## Motivations
In the activitiesView service with `watched_group_id`, most groups does not any activities returned as it requires both the current user and the watched group to have access to the item. The need for the watched group to have access is not that important as group members may have access by other means (for instance, "all users" group).

## Subtasks
Remove the requirement for the watched group to be allowed to view an item for it to be listed in the `activitiesView` service with `watched_group_id` parameter.

Same for skillsView

Same for itemNavigationView

### PR DESCRIPTION
**Pull request by zenovich**
Fixes #738

### CODE REVIEW

**Review by smadbe — CHANGES_REQUESTED** [review]
I would expect a spec/doc change to explain the two changes (perm required, perm returned)

**Review by smadbe — APPROVED** [review]
actually I can approve, but please do the doc changes.

**Comment by zenovich** [comment]
> I would expect a spec/doc change to explain the two changes (perm required, perm returned)

fixed

File: app/api/currentuser/get_group_activities.go [hunk 1]
```diff
@@ -92,8 +92,10 @@ type rootItem struct {
 // description:
 //   If `{watched_group_id}` is not given, the service returns the list of root activities of the groups the current user
 //   (or `{as_team_id}`) belongs to.
-//   Otherwise, the service returns the list of root activities of all ancestor groups of the watched group which are also
+//   Otherwise, the service returns the list of root activities (visible to the current user or `{as_team_id}`)
+//   of all ancestor groups of the watched group which are also
 //   ancestors or descendants of at least one group that the current user manages.
```
Comment (smadbe) [inline app/api/currentuser/get_group_activities.go:97]: Reading  "of all ancestor groups of the watched group which are also ancestors or descendants of at least one group that the current user manages." -> but as the watched group is manage, all ancestors of the watched group are "ancestors or descendants of at least one group that the current user manages". Am I wrong?

Comment (zenovich) [reply app/api/currentuser/get_group_activities.go:97]: Not really.
```
A   B
  \ /
   C
```
If the current user is a manager of B, and the watched group is C, then the current user can manage C, but the root activity of A will not be displayed although it is an ancestor of the watched group.

Comment (smadbe) [reply app/api/currentuser/get_group_activities.go:97]: You should use "explicitly manage" and "implicitly manage". 
So your "at least one group that the current user manages" means "explicitly manage" while "the user should ba a manager of the group" two sentences below mean "implicitly manage"?

Comment (zenovich) [reply app/api/currentuser/get_group_activities.go:97]: Fixed in #744

**Pull request merged**